Optik/Classes/Optik.swift

```diff
@@ -40,10 +44,12 @@ public func imageViewerWithImages(images: [UIImage],
  */
 public func imageViewerWithURLs(urls: [NSURL],
                                 imageDownloader: ImageDownloader,
+                                imageDisplayIndex: Int = 0,
```

Comment:
we should make this second argument of the function just like the other function so that the caller can always expect `imageDisplayIndex` to come after the array. less thinking required once you are familiar with the API.